### PR TITLE
Waitfor22 aws

### PR DESCRIPTION
**Pull request by enxebre**

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -2,9 +2,21 @@
...
+      wait_for:
+        port: 22
+        host: "{{ inventory_hostname }}"
+        delay: 10
+        timeout: 60
+      delegate_to: "{{ nat_ip }}"
+      sudo: False
+      when: PROVIDER == 'aws' or PROVIDER == 'AWS'
+
+    - name: Wait for port 22 to become available from local sever.
```
Comment (tayzlor) [inline site.yml]: *server*

**Comment by wallies** [comment]
Is there docs referring to NAT

**Comment by tayzlor** [comment]
don't think there's docs but theres an architecture diagram.

Otherwise LGTM.

**Comment by tayzlor** [comment]
we could probably update the diagram in a follow-up PR - its changing anyway as part of the multi-az

**Pull request merged**